docs/docs/compat.md

```diff
@@ -9,7 +9,17 @@
 ### Unsupported
 
 - Mysql 5.7 is untested with Maxwell, and in particular GTID replication is unsupported as of yet.
-- `binlog_row_image=MINIMAL` is not supported and will break Maxwell in a variety of amusing ways.
+
+### binlog_row_image=MINIMAL
```

Comment:
How does binlog_row_image=MINIMAL work with log compaction? Or, does it not work with log compaction? I guess log compaction isn't technically a Maxwell concern, it's a Kafka concern, so it's up to the user if they care about it.